### PR TITLE
Two monthly meetings for WG batch targeting different timezones

### LINKED ISSUE #7659: Add monthly WG-Batch meeting for US-west+APAC
### Describe the issue

We currently have a meeting every other Thursday at 3PM UTC, which is within working hours for European and East North American cities, where the coordinators are.
However, the slot is too late for APAC and too early for West NA.

Given the interest for people from these places to participate, we can consider adding a monthly meeting outside of the working hours of the existing coordinators.

An initial survey was sent by @vsoch https://doodle.com/meeting/participate/id/e02w44Lb, but it didn't gather any votes from APAC. I'm wondering whether there was enough outreach or the proposed slots are too early.

### PR DESCRIPTION
**Pull request by alculquicondor**
- Move existing meeting one hour earlier and make it monthly
- Introduce a new meeting targeting 3PM PT.

We heard feedback from members in APAC that it was acceptable for them to join late meetings, while it was not possible for them to join early meetings.

3pm was the overall preferred time for the US West Coast https://doodle.com/meeting/participate/id/eEKlnM0b
This time is also accessible for people in Australia.

Once this PR merges, we can update the calendar and doc.

**Which issue(s) this PR fixes**:

Fixes #7659

### CODE REVIEW

**Review by swatisehgal — APPROVED** [review]
/lgtm
/approve

Thanks for helping out with this!

**Comment by vsoch** [comment]
I'm so excited to see this move through - thank you for your support @alculquicondor ! :raised_hands:

File: sig-list.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ When the need arises, a [new SIG can be created](sig-wg-lifecycle.md)
...
+|[Batch](wg-batch/README.md)|[batch](https://github.com/kubernetes/kubernetes/labels/wg%2Fbatch)|* Apps<br>* Autoscaling<br>* Node<br>* Scheduling<br>|* [Aldo Culquicondor](https://github.com/alculquicondor), Google<br>* [Marcin Wielgus](https://github.com/mwielgus), Google<br>* [Maciej Szulik](https://github.com/soltysh), Red Hat<br>* [Swati Sehgal](https://github.com/swatisehgal), Red Hat<br>|* [Slack](https://kubernetes.slack.com/messages/wg-batch)<br>* [Mailing List](https://groups.google.com/a/kubernetes.io/g/wg-batch)|* Regular Meeting ([Calendar](https://calendar.google.com/calendar/embed?src=8ulop9k0jfpuo0t7kp8d9ubtj4%40group.calendar.google.com)): [Thursdays (starting February 15th 2024)s at 3PM CET (Central European Time) (monthly)](https://zoom.us/j/98329676612?pwd=c0N2bVV1aTh2VzltckdXSitaZXBKQT09)<br>* Regular Meeting ([Calendar](https://calendar.google.com/calendar/embed?src=8ulop9k0jfpuo0t7kp8d9ubtj4%40group.calendar.google.com)): [Thursdays (starting February 1st 2024)s at 3PM PT (Pacific Time) (monthly)](https://zoom.us/j/98329676612?pwd=c0N2bVV1aTh2VzltckdXSitaZXBKQT09)<br>
```
Comment (kerthcet) [inline sig-list.md:65]: For confirmation, both monthly not bi-weekly. If monthly, should we clarify which week?

Comment (alculquicondor) [reply sig-list.md:65]: They are both monthly, yes.

You can see that one says starting Feb 1st, and the other Feb 15th.

For details, people can refer to the calendar.

Comment (alculquicondor) [reply sig-list.md:65]: I updated the calendar just now.

**Pull request merged**